.github/workflows/ci.yml

```diff
@@ -45,13 +45,12 @@ jobs:
             experimental: true
     runs-on: ${{ matrix.os }}
     continue-on-error: ${{ matrix.experimental }}
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
+      cancel-in-progress: true
```

Comment:
I'm seeing the presubmit for this PR is cancelling all but one of the actions with the message

> Canceling since a higher priority waiting request for 'CI-refs/pull/1016/merge' exists

I think maybe the issue is that the actions for different JDK versions are using the same concurrency group, and then `cancel-in-progress` cancels all but one of the them?

Reply:
let me switch this to a different PR, can you also attach the run logs for me to debug?